.gitignore

```diff
@@ -1,5 +1,4 @@
 # See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
-.env
```

Comment:
Why do we want to not ignore .env? 😅

Reply:
for some reason, every time I commit my `.env` is cleared out anyway (any idea? some pre-commit script?). If we avoid clearing out `.env` (preferable) then it makes sense to add `.env` in gitignore